contrib/mesos/pkg/offers/offers.go

```diff
@@ -441,7 +441,7 @@ func (s *offerStorage) ageOffers() {
 }
 
 func (s *offerStorage) nextListener() *offerListener {
-	obj := s.listeners.Pop()
+	obj := s.listeners.Pop(nil)
```

Comment:
Using the symbol `queue.WithoutCancel` would be much better IMO than just `nil`.